sockeye/arguments.py

```diff
@@ -369,6 +369,11 @@ def add_training_args(params):
                               default=13,
                               help='Random seed. Default: %(default)s.')
 
+    train_params.add_argument('--keep-last-params',
+                              type=int,
+                              default=20,
```

Comment:
would it make sense to set the default to -1 to stay backwards compatible? If not I'd say we should set it to something higher than 20, as with values of num_not_improved as high as 16 (which we probably should also set to as the default in arguments.py), one would have only 4 checkpoints before the last checkpoint that improved. This would then hurt the 'best' strategy of average.